modules/rosa-hcp-firewall-prerequisites.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * rosa_planning/rosa-sts-aws-prereqs.adoc
+// * rosa_planning/rosa-hcp-prereqs.adoc
+
+[id="rosa-hcp-firewall-prerequisites_{context}"]
+// Conditionals are to change the title when displayed on the rosa-sts-aws-prereqs page
+ifeval::["{context}" == "rosa-sts-aws-prereqs"]
+= ROSA with HCP
+endif::[]
+ifeval::["{context}" != "rosa-sts-aws-prereqs"]
+= AWS firewall prerequisites
+
+This section provides the necessary details that enable you to control egress traffic from your {product-title} with Hosted Control Planes cluster. If you are using a firewall to control egress traffic, you must configure your firewall to grant access to the domain and port combinations below. {product-title} with Hosted Control Planes requires this access to provide a fully managed OpenShift service.
+endif::[]
+
+.Requirements
```

Comment:
s/.Requirements/.Prerequisites

Per the documentation guidelines these need to be prerequisites.